react-native/react/engine/index.js

```diff
@@ -118,8 +118,10 @@ class Engine {
   _generalIncomingRpc (method, param, response) {
     (this.generalListeners[method] || []).forEach(listener => {
       // TODO(mm) does it make sense to ever pass the response?
-      // It'll be weird if there are multiple listeners
-      listener(param)
+      // It'll be weird if there are multiple listeners.
+      // We should get the response callback to work the first
+      // time through and fail the second time.
+      listener(param, response)
```

Comment:
Is this the right idea, changing the response function to fire once and fail thereafter, so that we don't end up trying to send multiple replies if (for whatever reason) we've ended up with multiple listeners?